addons/mail/static/src/discuss/call/common/ptt_extension_service.js

```diff
@@ -5,6 +5,13 @@ import { browser } from "@web/core/browser/browser";
 import { registry } from "@web/core/registry";
 import { _t } from "@web/core/l10n/translation";
 
+/** In object so it's patchable */
+export const pttExtensionServiceInternal = {
+    onAnswerIsEnabled(pttService) {
+        pttService.isEnabled = true;
+    },
+};
+
 export const pttExtensionHookService = {
     start(env) {
         const INITIAL_RELEASE_TIMEOUT = 500;
```

Comment:
While we're at it, I recently noticed that 500ms wasn't enough to prevent the initial flicker. I'd increase it to 750ms to avoid the issue. I observed that the delay between the first two events was ~650ms, and the following events were spaced by ~150ms.